scripts/process/scaleStates.R

```diff
@@ -0,0 +1,41 @@
+#scale states according to water use
+# two different sized states with equal water use should have the 
+# same area
+library(rgeos)
+library(dplyr)
+
+process.scaleStates <- function(viz){
+  #could put these in yaml too?
+  coreCats <- c("Thermoelectric", "Irrigation", "Public Supply", "Industrial")
+  
+  #read sp, wuClean
+  statePoly <- readData(viz[['depends']]$stateMap)
+  wuClean <- readData(viz[['depends']]$wuClean)
+  
+  wuClean$state_name <- tolower(wuClean$state_name)
+  #get areas, join to wuClean
+  areas <- data.frame(gArea(statePoly, byid = TRUE), stringsAsFactors = FALSE)
+  areas$state_name <- rownames(areas)
+  areas["district of columbia",][2] <- "dist. of columbia"
```

Comment:
Is that name consistent with the name in the `maps` pkg that is being used as the id reference in the svg?